docs/content/latest/admin/yb-admin.md

```diff
@@ -45,14 +45,14 @@ $ ./bin/yb-admin --help
 - [Table](#table-commands)
 - [Backup and snapshot](#backup-and-snapshot-commands)
 - [Deployment topology](#deployment-topology-commands)
-  - [Multi-zone and multi-region](#multi-zone-and-multi-region-commands)
-  - [Master-follower](#master-follower-commands)
-  - [Read replica](#read-replica-commands)
+    - [Multi-zone and multi-region](#multi-zone-and-multi-region-deployment-commands)
+    - [Master-follower](#master-follower-deployment-commands)
+    - [Read replica](#read-replica-deployment-commands)
+- [Security](#security-commands)
+    - [Encryption at rest](#encryption-at-rest-commands)
```

Comment:
oh did these need 4 spacing?

Reply:
The four-space indent for sub-bullets is a good idea in markdown, as there have been rendering bugs in the past with smaller indents. Markdownlint catches these, and I fix them as I find them...

Reply:
Though if two-space indenting is a lot easier — which it is — and we're not seeing any rendering bugs, maybe I should change that linter rule to 2 spaces?

Reply:
oh I'm fine either way, was just curious

out of scope for this PR: is markdownlint something we can install on all machines so all docs folks can get it? and do you manually run it yourself? for the normal coding flow via phabricator, we run linters automatically, before putting up diffs, wonder if we can do the same for docs PRs?

Reply:
It sure is, and I've got a configuration file in the repo for it already. Just not ready to run it as a PR action yet. :)